packages/forklift-console-plugin/src/modules/Plans/views/create/PlanCreatePage.tsx

```diff
@@ -59,6 +59,8 @@ export const PlanCreatePage: React.FC<{ namespace: string }> = ({ namespace }) =
   });
   useSaveEffect(state, dispatch);
 
+  const isFirstStepValid = validateSourceProviderStep(state, filterState);
```

Comment:
thanks, could u add useMemo here? there are network calls and other props that will make it recalc

Reply:
Updated.